style.css

```diff
@@ -1,12 +1,61 @@
 body {
 	margin: 20px;
 	font-family: 'Roboto', sans-serif;
+	
 }
 
 header {
+	width: 100%;
     text-align: center;
+    background-position: 90% 100%;
+    background-repeat: no-repeat;
+    
 }
 
+.head_main{
+	background-color: #0996f8;
+	height: 250px;
+}
+
+#logo {
+	height: 100px;
+    padding: 15px;
+    width: 200px;
+}
+
+#logo img {
+	height: 100px;
+    float: left;
```

Comment:
try flex-box, instead of floats. This will help you deal with mobile layout too.